browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -644,31 +651,46 @@ void RewardsDOMHandler::OnAutoContributePropsReady(
       props->reconcile_stamp,
       props->contribution_non_verified,
       props->contribution_min_visits,
+      false,
       base::Bind(&RewardsDOMHandler::OnContentSiteList,
                  weak_factory_.GetWeakPtr()));
 }
 
+void RewardsDOMHandler::OnAutoContributePropsReadyExcluded(
+    std::unique_ptr<brave_rewards::AutoContributeProps> props) {
+  rewards_service_->GetContentSiteList(
+      0,
+      0,
+      props->contribution_min_time,
+      props->reconcile_stamp,
+      props->contribution_non_verified,
+      props->contribution_min_visits,
+      true,
+      base::Bind(&RewardsDOMHandler::OnExcludedSiteList,
+                 weak_factory_.GetWeakPtr()));
+}
+
 void RewardsDOMHandler::OnContentSiteUpdated(
     brave_rewards::RewardsService* rewards_service) {
   rewards_service_->GetAutoContributeProps(
       base::Bind(&RewardsDOMHandler::OnAutoContributePropsReady,
         weak_factory_.GetWeakPtr()));
 }
 
-void RewardsDOMHandler::OnGetExcludedPublishersNumber(uint32_t num) {
-  if (web_ui()->CanCallJavascript()) {
-    web_ui()->CallJavascriptFunctionUnsafe("brave_rewards.excludedNumber",
-        base::Value(std::to_string(num)));
-  }
+void RewardsDOMHandler::GetExcludedSites(const base::ListValue* args) {
+  rewards_service_->GetAutoContributeProps(
+      base::Bind(&RewardsDOMHandler::OnAutoContributePropsReadyExcluded,
+        weak_factory_.GetWeakPtr()));
 }
+
 void RewardsDOMHandler::OnExcludedSitesChanged(
     brave_rewards::RewardsService* rewards_service,
     std::string publisher_id,
     bool excluded) {
-  if (rewards_service_)
-    rewards_service_->GetExcludedPublishersNumber(
-        base::Bind(&RewardsDOMHandler::OnGetExcludedPublishersNumber,
-                   weak_factory_.GetWeakPtr()));
+  if (rewards_service_) {
+    base::ListValue args;
+    GetExcludedSites(&args);
```

Comment:
this change should be propagated to JS and then JS should call `GetExcludedSites`

Reply:
done